src/azure-cli/azure/cli/command_modules/botservice/custom.py

```diff
@@ -100,7 +100,7 @@ def __handle_failed_name_check(name_response, cmd, client, resource_group_name,
 
 
 def create(cmd, client, resource_group_name, resource_name, kind, msa_app_id, password=None, language=None,  # pylint: disable=too-many-locals, too-many-statements, inconsistent-return-statements
-           description=None, display_name=None, endpoint=None, tags=None, location='global',
+           description=None, display_name=None, endpoint=None, tags=None, location='southcentralus',
```

Comment:
Why do we need to modify the default location here? Will this cause breaking change?

Reply:
@zhoxing-ms  thanks for pointing this out - can we have someone from BOT team help here - I am not sure how global worked since "global" is not a supported location on App Service. I read this as they are trying to default a location, if one is not provided.

Reply:
@munozemilio Could you please take a look at this question?

Reply:
@munozemilio any update here?

Reply:
@panchagnula Is it necessary for us to revise here? If so, why?

Reply:
yeah because global is not a location that is supported - so Webapps support with bot apps will fail

Reply:
@panchagnula In fact, my question is whether we need to include the modification of bot service code in this hot fix, or we can make this code change in the later sprint

Reply:
@yonzhan if this Q isn't resolved in time go-ahead with current cloud shell release. I will work offline with you if there is a possibility to take this as hotfix, instead waiting for a next milestone release.

Reply:
@jocrafts 'global' is not a supported location name in App services, so I am not sure how this change worked? Can you comment - since this is blocking current release for SDK update for websites extension (see this was a recent change made by you https://github.com/Azure/azure-cli/pull/20716)

Reply:
@zhoxing-ms I noticed the change that made here on BotService - did not re-record any tests , hence this issue was never originally found https://github.com/Azure/azure-cli/pull/20716/ I can go-ahead remove this in my PR & disable the failing bot service tests.